src/content/docs/mobile-monitoring/new-relic-mobile-xamarin/monitor-your-xamarin-application.mdx

```diff
@@ -10,6 +10,8 @@ metaDescription: "New Relic's Xamarin agent for Android and iOS: Features, compa
 
 import mobileFlutterSummary from 'images/mobile_screenshot-full_flutter-summary.webp'
 
+import mobileMobileAppToken from 'images/mobile_screenshot-crop_mobile-app-token.webp'
```

Comment:
@ally-sassman I'm not seeing an instance of this image down in the body of the page. Is it possible that this image was meant for this file instead: src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/monitor-your-react-native-application.mdx